### ISSUE TITLE
'conda clean --lock' is suggested but does not exist

### ISSUE THREAD

**Issue by basnijholt** [mamba-org/mamba#1200]
I got this error in my CI logs:
```bash
Exceeded max retries, giving up
LockError: 
LOCKERROR: It looks like conda is already doing something.
The lock ['/mnt/resource/mambaforge/pkgs/cache.pid22991.conda_lock'] was found. Wait for it to finish before continuing.
If you are sure that conda is not running, remove it and try again.
You can also use: $ conda clean --lock
```
which occurs because a CI pipeline got canceled in a previous run and now the lock exists in that state.

Then when I added the suggested command to the CI's script, I get:
```bash
usage: conda [-h] [-V] command ...
conda: error: unrecognized arguments: --lock
```

The latest mamba `4.10.3-6` is installed via
```bash
wget --quiet https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-Linux-x86_64.sh
bash Mambaforge-Linux-x86_64.sh -b 
```

**Comment by wolfv** [comment]
@adriendelsalle is working on improved locks just now. Maybe we should use ours in pure mamba (from python) as well (as opposed to the lock files from conda which we're currently using).

What do you think @adriendelsalle?

**Comment by adriendelsalle** [comment]
Agree. We should also support CLI flags to clean lock files micromamba.
We have to take care of cleaning the lib mamba locks in mamba if we switch the implementation from conda to libmamba

**Comment by hmaarrfk** [comment]
Just cross referencing an other post of mine that I wrote hitting the same issue.

I feel that the choice of locks is rather poor and doesn't lend itself well with current use-cases:

1. The conda processing taking alot of time, and either crashes, or the user gets frustrated and presses ctrl+c to kill it.

https://github.com/mamba-org/mamba/issues/1205#issuecomment-945129653
